src/components/dashboard/InfiniteList.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from 'react';
+import InfiniteScroll from 'react-infinite-scroller';
+
+import Loading from './../Loading';
+
+class InfiniteList extends Component {
+    render() {
+        return (
+            <InfiniteScroll
+                pageStart={0}
+                loadMore={this.props.update}
+                hasMore={this.props.items.next}
+                useWindow={false}
+                threshold={-256}
+                loader={<Loading fetching={this.props.items.fetching} length={this.props.items.length}/>}
```

Comment:
since `this.props.items` isn't used here, should we pass `length`, `fetching`, and `next` instead of the full `items` object?